### ISSUE TITLE
Plot # of users

### ISSUE THREAD

**Issue by kappa90** [PostHog/posthog#3334]
## Is your feature request related to a problem?

Currently plots are only by action/events.
I would like to plot the cumulative number of identified users, or the cumulative number of users inside a cohort.

## Describe the solution you'd like

Instead of having to select an event as the base metric of the plot, I would like to select the cumulative sum of users inside a cohort, or of all identified/unidentified users.

## Describe alternatives you've considered

We currently use an opt-in event at signup with a cumulative plot to track total users, but it only starts from the day we started using posthog, so we don't have that event for pre-existing users.

We are also not able to the total number of users in a cohort if that is not correlated to an event. E.g. we would like to see all users on a specific OS over time.

**Comment by yakkomajuri** [comment]
Hey! Here's what you can do:

1. Next to the event selector you will see 'Total Volume'. Change that to 'Active Users'
2. On top of the graph you will see 'Linear', change this to 'Cumulative'
3. Then add a filter to the entire graph by cohort
4. And finally, add an event that every user has (like a pageview) as the matching event (if you don't have such an event, you can create an action matching multiple events)

**Comment by yakkomajuri** [comment]
Here's an example:

<img width="1440" alt="Screenshot 2021-02-15 at 09 01 19" src="https://user-images.githubusercontent.com/38760734/107925469-66fdeb00-6f6c-11eb-9106-f02c4ff6adab.png">

**Comment by kappa90** [comment]
Thanks for the reply!

I might not have explained myself correctly. What I want to plot is not the number of cumulative pageviews filtered by a cohort. I would like to plot how big the cohort is over time, regardless of events.

If instead you are actually suggesting a solution to what I'm asking, I think it's not working, see below:

15 users in this test cohort
![image](https://user-images.githubusercontent.com/7335343/107926574-2d31e200-6f76-11eb-975b-8cea32060ef5.png)

Almost double if I plot it by pageview/active users
![image](https://user-images.githubusercontent.com/7335343/107926662-46d32980-6f76-11eb-8f5f-9e303286e410.png)

Thanks again!

**Comment by yakkomajuri** [comment]
Yes, so I understood you're not looking to track pageviews. What I was doing there is using pageview as a proxy for an getting all users in the cohort.

Essentially, active users tags one event per user for each period (daily, weekly, monthly, etc.).

If you want to track size of a cohort over time, you could just leave it at linear and use a larger time frame (e.g. monthly). This will give you all the unique users (not events) that did the certain event over that period. If the event is common enough, you're essentially tracking the cohort size, although I understand this is a bit hacky.

**Comment by kappa90** [comment]
I see, that gives me the correct number over the last month, but this means if we go with this for more than a month, the metric will again fail because it will only account for the last month, not for the actual cohort size.

![image](https://user-images.githubusercontent.com/7335343/107928447-9f0b2b00-6f78-11eb-8fe6-f17e16bf7b31.png)

Another thing since you mentioned being able to create actions to track multiple events: would it make sense for you to add a wildcard "any event", in this way we could track active users who trigger any event but don't trigger a pageview, because that's not working if the user has an open tab over multiple days and doesn't reload.

Thanks again!

**Comment by jdagdelen** [comment]
Can we please build something for this? Log a cohort's size every day, week, month. We have cohorts like "Have used the app more than 3 days this week" and we want to track how we are doing relative to previous months.

**Comment by mason-os** [comment]
I've also been looking for an answer to this question-- we have a power user cohort and I'd love to be able to track the size over time. I'm using it as an analog for retention though, as we'd like to define activation and retention as an event happening a certain number of times rather than 1 time/multiple times.

**Comment by davidtopf** [comment]
I need this too. I want to track my paying users, which may or may not have had a pageview event in the last month or two. They may not have interacted with the app at all.
